cryptofeed/backends/quest.py

```diff
@@ -68,11 +74,12 @@ def __init__(self, *args, depth=10, **kwargs):
         super().__init__(*args, **kwargs)
         self.depth = depth
 
-    async def __call__(self, book, receipt_timestamp: float):
+    async def __call__(self, book, receipt_timestamp_int: float):
```

Comment:
the value passed in here to receipt timestamp is definitely a float, so curious why you changed the name?

Reply:
typo, fixed